docs/src/image_loading.md

```diff
@@ -0,0 +1,203 @@
+# Image Loading
+
+## Overview
+
+The Image Loading module is a component of the MCU Runtime SDK designed for managing SOC images stored in flash memory. This module provides APIs for:
+
+- Detecting SOC images availability in the [flash storage](./image_loading.md).
+- Parsing the [SOC manifest](https://github.com/chipsalliance/caliptra-sw/blob/main/auth-manifest/README.md).
+- Loading SOC images to a target component.
+
+The diagram below illustrates an **example** how the Caliptra subsystem, integrated within the SOC, is used to load SOC images to vendor components. In this example, these are the 2 Vendor CPUs and their corresponding RAMs.
+
+<p align="center">
+    <img src="images/image_loading_sample.svg" alt="flash_config" width="50%">
+</p>
+
+Pre-image loading state:
+
+* Caliptra MCU Runtime Firmware (MCU RT) is running in the MCU.
+* The SOC target components (Vendor CPU 1 and 2) are held in reset.
+
+Image Loading Steps:
+
+1. MCU RT checks for available SOC images in the [flash storage](./image_loading.md).
+2. MCU RT authorizes the [SOC manifest](https://github.com/chipsalliance/caliptra-sw/blob/main/auth-manifest/README.md) and the SOC images by following steps described in [Manifest-Based Authorization](https://github.com/chipsalliance/caliptra-sw/blob/main-2.x/runtime/README.md#manifest-based-image-authorization-new-in-12) section of the Caliptra Runtime Specification.
+3. If the SOC images are authorized and verified, MCU RT releases the reset of the target components.
+
+## Architecture
+
+The following diagram presents the software stack architecture where the Image Loading module resides.
+
+<p align="left">
+    <img src="images/image_loading_sw_stack.svg" alt="sw_stack" width="30%">
```

Comment:
This is better. A few tips:

* Make it larger so that you can make all the text horizontal.
* Try to match the color scheme in the overview doc: gray for apps, oranges for handlers and libraries, soft yellow for SDKs, soft blue for the kernel 

I'll make a note to myself to try to formalize some of this in a style guide.

Reply:
Updated the image. You can find the raw image in page-12 of this Visio file: https://microsoft-my.sharepoint.com/:u:/p/marcovisaya/EcaNju7LbulCjYiP533qQzIB2ZruXr3RtT_XBiOwlOCA9Q?e=DAdazr